### ISSUE TITLE
Use tags to group Guru content for trainees

### ISSUE THREAD

**Issue by astronotjake** [Shopify/km#14]
@Shaylene mentioned a neat idea about how to use tags for training purposes. 

Now that KM is actually controlling tags and their use, Training could look at using the tags to group content for their trainees more easily rather than just linking to the content from Bridge. 

As an example, all the content in Guru which would be used in Week 1 could be tagged `#week 1` so trainees can quickly find the content while getting used to the search style we want Gurus to get used to using once tags are fully rolled out. 

This could also be done by building out boards, as they are more easily browsed and can be filtered in search but tags will be part of the main workflow of Gurus on the job so there's a bit of a trade off. 

CC: @Llt2 since you weren't around for the convo between Shay and I.

**Comment by Shaylene** [comment]
nice summary Jake!

**Comment by tkirrane** [comment]
I like it!

**Comment by astronotjake** [comment]
From Alex Roodman:

"oh wow that’s really cool.

Hmm, okay I just chatted with a few from my crew and while I can see how this would make the lives of Gurus easier, the exercise of bookmarking resources when the come across them in training, and searching them out when they have questions is an important part of training the resourcefulness that’s so key in onboarding.

I’m not sure how often trainees would search for #week 1 in Guru (I think they’re more likely to search for the specific card), so it’s likely not worth our efforts to set this up and maintain it at the rate that our content evolves. For now. It’s really neat to know that this is an option for any future plans."

So this doesn't sounds like something Training is interested in doing at the moment, but they'll be thinking of ways we can implement tags for them in the future. Closing this issue since it's a non-starter for now.
